en/docs/guides/howtoguides/s3-events-via-sqs-listener.md

```
@@ -0,0 +1,355 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Process S3 File Events with the SQS Listener
+
+**What you'll build:** A Ballerina integration that reacts every time a file is uploaded to an S3 bucket. S3 pushes a notification to an SQS queue; the `aws.sqs` listener picks it up and your service logs the bucket name, object key, and file size without polling S3 directly.
+
+## How it works
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)